emsdk_manifest.json

```diff
@@ -466,6 +466,29 @@
     "windows_url": "mingw_7.1.0_64bit.zip",
     "activated_cfg": "MINGW_ROOT='%installation_dir%'",
     "activated_path": "%installation_dir%/bin"
+  },
+  {
+    "id": "ninja",
+    "version": "git-release",
+    "bitness": 64,
+    "url": "https://github.com/ninja-build/ninja.git",
+    "git_branch": "release",
+    "activated_cfg": "NINJA=%installation_dir%/bin",
+    "activated_path": "%installation_dir%/bin",
+    "cmake_build_type": "Release",
+    "custom_install_script": "build_ninja"
```

Comment:
Mainly because of being able to change up ninja code in a custom branch, because of getting direct Apple M1 Mac support this way, and to get backwards compatibility support against whichever Xcode SDK and Windows SDK that I compile for myself. (I don't know the guaranteed min OS versions of the precompiled releases).

Reply:
Should we at least try it?  or at least have "download binary" as the default way to download ninja?  (then only folks with unusual OS/hardware need to build from source, like we do for llvm)

Reply:
Do you have have patches to ninja planned?

Reply:
I would not oppose adding prebuilt ninja as well, though I am not keen to work on that at the moment. Fortunately emsdk does allow a good structure to have both.

I do plan to change ninja to fix stdout printing issues on Windows. When one uses ninja, they lose colored output for error messages and all stdout gets needlessly buffered, resulting in EMCC_DEBUG being uncomfortable to use on Windows.

Reply:
I would prefer to simply use ninja pre-built bit I'm OK landing this as is.  

Regarding the specifics of the ninja buffering issue.  Are you talking about how the buffering causes the tools to skip adding color to their output?  I agree it would be nice to find a fix for this, but I also really like how ninja buffers command and output rather than the nasty interleaving one gets with `make VERVOSE=1 -jN`.